strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/StoragesConfigurationController.java

```diff
@@ -332,4 +347,51 @@ public ResponseEntity removeRepository(@RepositoryMapping Repository repository,
         }
     }
 
+    @ApiOperation(value = "Deletes recursively the content of path located under the defined storageId and repositoryId.")
+    @ApiResponses(value = { @ApiResponse(code = 200, message = "The repository path was deleted successfully."),
+                            @ApiResponse(code = 404, message = "The repository path cannot be deleted because storageId,repositoryId,or path was not found!"),
+                            @ApiResponse(code = 500, message = "Failed to remove the repository path") })
+    @PreAuthorize("hasAuthority('CONFIGURATION_DELETE_REPOSITORY_PATH')")
+    @DeleteMapping(value = "/delete", consumes = MediaType.APPLICATION_JSON_VALUE, produces = { MediaType.TEXT_PLAIN_VALUE,
+                                                                                                MediaType.APPLICATION_JSON_VALUE })
+    public ResponseEntity removeRepositoryPath(@ApiParam(value = "The path object", required = true)
+                                               @RequestBody @Validated({ Default.class,
+                                                                         ProxyConfigurationFormChecks.class})
+                                               PathForm pathForm,
+                                               BindingResult bindingResult,
+                                               @RequestHeader(HttpHeaders.ACCEPT) String accept)
+    {
+
+        if(bindingResult.hasErrors())
+        {
+            return getFailedResponseEntity(HttpStatus.NOT_FOUND, FAILED_REPOSITORY_PATH_REMOVAL_FORM_ERROR, accept);
+        }
+
+        try
+        {
+            final RepositoryPath repositoryPath = repositoryPathResolver.resolve(pathForm.getStorageId(),
+                                                                                 pathForm.getRepositoryId(),
+                                                                                 pathForm.getPath());
+
+            if (!Files.exists(repositoryPath))
+            {
+                return getFailedResponseEntity(HttpStatus.NOT_FOUND, FAILED_REPOSITORY_PATH_REMOVAL_PATH_NOT_FOUND, accept);
+            }
+
+            artifactManagementService.delete(repositoryPath, pathForm.isForce());
+
+            logger.info("Deleting Path {}:{}/{}...", pathForm.getStorageId(), pathForm.getRepositoryId(), pathForm.getPath());
+
+            return getSuccessfulResponseEntity(SUCCESSFUL_REPOSITORY_PATH_REMOVAL, accept);
+        }
+        catch (NullPointerException | RepositoryRelativePathConstructionException e)
+        {
+            return getFailedResponseEntity(HttpStatus.NOT_FOUND, FAILED_REPOSITORY_PATH_REMOVAL_FORM_ERROR, accept);
```

Comment:
This message is wrong. Here you should use `FAILED_REPOSITORY_PATH_REMOVAL_PATH_NOT_FOUND` or another message more specific.